impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js

```diff
@@ -0,0 +1,601 @@
+
+            legendAlign: "center",
+
+            // Plot
+            activeSliceRadius: 0,
+
+            // . slice
+            slice_ibits: 0,
+            slice_imask: "ShowsActivity",
+            slice_fillStyle: function() {
+              var c = this.delegate();
+              return this.finished(c && this.scene.isActive ? c.alpha(0.5) : c);
+            }
+          }
+        }
+      },
+
+      // Donut
+      {
+        select: {
+          priority: -1,
+          // application: "pentaho-det",
+          type: "pentaho/ccc/visual/donut"
+        },
+        apply: {
+          extension: {
+            valuesVisible: false,
+            slice_lineWidth: 0,
+            slice_innerRadiusEx: "60%"
+          }
+        }
+      },
+
+      // Heat Grid
+      {
+        select: {
+          priority: -1,
+          // application: "pentaho-det",
+          type: "pentaho/ccc/visual/heatGrid"
+        },
+        apply: {
+          extension: {
+            // Don't use composite/hierarchical axes
+            axisComposite: false,
+
+            // ---
+
+            // . rule
+            axisRule_lineWidth: 0,
+
+            // . grid
+            orthoAxisGrid: false,
+            baseAxisGrid:  false,
+
+            axisBandSpacing: 5, // white border or transparent ?
+
+            // X
+            xAxisPosition: "top",
+            xAxisSizeMax: 80,
+            xAxisBandSizeMin: 30, // make equal to the inherited yAxisBandSizeMin: 30,
+
+            // Y
+            yAxisSizeMax: 80, // shouldn't it be: maxHorizontalTextWidth ??
+            yAxisBandSizeMin: 30,
+
+            // . label
+            xAxisOverlappedLabelsMode: "rotatethenhide",
+            xAxisLabelRotationDirection: "counterclockwise",
+            xAxisLabelDesiredAngles: [0, 40 * (Math.PI / 180)],
+
+            // overwrite color scale
+            colors: ["#8eb7d9", "#002f55"],
+
+            // . dot
+            dot_ibits: 0,
+            dot_imask: "ShowsActivity",
+            dot_lineWidth: 0,
+            dot_fillStyle: function() {
+              var c = this.delegate();
+              return this.finished(c && this.scene.isActive ? c.alpha(0.5) : c);
+            }
+          }
+        }
+      }
+    ]
+  };
+
+  function getNumberFormatter(precision, base) {
+    if (!pvc) {
+      pvc = require("cdf/lib/CCC/pvc");
+    }
+
+    var useAbrev = (base >= 1000);
+    var key = useAbrev + "|" + precision;
+    var numberFormat = numberFormatCache[key];
+    if (!numberFormat) {
+      // #,0 A
+      // #,0.0 A
+      // #,0.00 A
+      var depPlacesMask = precision ? ("." + new Array(precision + 1).join("0")) : "";
+      var mask = "#,0" + depPlacesMask + (useAbrev ? " A" : "");
+
+      numberFormat = pvc.data.numberFormat(mask);
+      numberFormat.style(numberStyle);
+      numberFormatCache[key] = numberFormat;
+    }
+
+    return numberFormat;
+  }
+
+  function configureVizAPIPalette() {
```

Comment:
This Viz palette stuff is already in the color palette registry and should be removed from here